ubuntu/debian/compat

```diff
@@ -1 +1 @@
-9
+11
```

Comment:
Per https://github.com/gazebo-release/sdformat14-release/pull/1#discussion_r1282520277 we should actually use 12 here.

Reply:
uh right, updated [5c81ae1](https://github.com/gazebo-release/sdformat13-release/pull/18/commits/5c81ae14c1f2a740dc64ce9e4bd45eb3f88a51ba)